presto-orc/src/main/java/com/facebook/presto/orc/metadata/DwrfMetadataWriter.java

```diff
@@ -252,6 +252,7 @@ public int writeStripeFooter(SliceOutput output, StripeFooter footer)
     {
         DwrfProto.Stream.Builder streamBuilder = DwrfProto.Stream.newBuilder()
                 .setColumn(stream.getColumn())
+                .setSequence(stream.getSequence())
```

Comment:
Can you add a check in the OrcMetdataWriter, the sequence is always DEFAULT_SEQUENCE_ID. ORC is open source and DWRF is Facebook internal version of it. ORC does not support sequence and if the sequence id is set to anything other than that, we have an issue.